routes/cube/deck.js

```diff
@@ -586,7 +586,7 @@ router.post('/submitdeck/:id', body('skipDeckbuilder').toBoolean(), async (req,
     // req.body contains a draft
     const draftid = req.body.body;
     const draft = await Draft.findById(draftid).lean();
-    const cube = await Cube.findOne(buildIdQuery(draft.cube));
+    const cube = await Cube.findOne(draft.cube);
```

Comment:
should this be changed to 'findById' like the other changes?

Reply:
They can be, but this is fine as well. The two are equivalent in these cases.